src/test/java/com/redislabs/redisai/DagTest.java

```diff
@@ -0,0 +1,161 @@
+package com.redislabs.redisai;
+
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+
+public class DagTest {
+  private final JedisPool pool = new JedisPool();
+  private final RedisAI client = new RedisAI(pool);
+
+  @Before
+  public void setUp() {
+    try (Jedis conn = pool.getResource()) {
+      conn.flushAll();
+    }
+  }
+  //
+  //    @After
+  //    public void tearDown(){
+  //        try (Jedis conn = pool.getResource()) {
+  //            conn.flushAll();
+  //        }
+  //    }
```

Comment:
no need for tearDown?

Reply:
missed uncommenting it. solved in latest commit